core/reactor_common.c

```diff
@@ -1101,7 +1102,20 @@ int _lf_schedule_at_tag(trigger_t* trigger, tag_t tag, lf_token_t* token) {
  * @return A handle to the event, or 0 if no new event was scheduled, or -1 for error.
  */
 trigger_handle_t _lf_schedule(trigger_t* trigger, interval_t extra_delay, lf_token_t* token) {
-    if (_lf_is_tag_after_stop_tag(current_tag)) {
+    
+    // Find the containing reactor
+    // FIXME: Is this safe? Will all the triggered reactions always be in the same containing reactor?
+    // FIXME: Is there a scenario where we want to use the local time at the "caller" rather than the reactor containing the action? 
```

Comment:
I think we *always* want to use the local time at the caller. I think the implementation will require adding a pointer to the self struct of the owner of the trigger_t struct.